cluster/gce/configure.sh

```diff
@@ -36,32 +38,143 @@ fetch_metadata() {
   fi
 }
 
-# DEPLOY_PATH is the gcs path where cri-containerd tarball is stored.
-DEPLOY_PATH=${DEPLOY_PATH:-"cri-containerd-release"}
+# fetch_env fetches environment variables from GCE metadata server
+# and generate a env file under ${CONTAINERD_HOME}. It assumes that
+# the environment variables in metadata are in yaml format.
+fetch_env() {
+  local -r env_name=$1
+  (
+    umask 700;
```

Comment:
why 700?

Reply:
Copied from kubernetes code. It seems to be changed recently.
https://github.com/kubernetes/kubernetes/pull/62301

Reply:
Changed to 077